### PR TITLE
Feat: [ASM] Adding link to customer list for navigating to customer360

### PR DESCRIPTION
**Pull request by scarai-sap**
See https://jira.tools.sap/browse/CXSPA-6858

### CODE REVIEW

File: feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts [hunk 1]
```diff
@@ -147,13 +148,44 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
...
+  protected showC360Dialog(customer: User | undefined): void {
+    const data = { customer: customer };
+    this.launchDialogService?.openDialogAndSubscribe(
+      LAUNCH_CALLER.ASM_CUSTOMER_360,
+      this.element,
+      data
+    );
+
+    this.subscription.add(
+      this.launchDialogService?.dialogClose
```
Comment (niehuayang) [inline feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts]: **?** is redundant.

File: feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts [hunk 2]
```diff
@@ -147,13 +148,44 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
...
             }
           }
         })
     );
     this.subscribeForDeeplink();
   }
 
+  protected showC360Dialog(customer: User | undefined): void {
+    const data = { customer: customer };
+    this.launchDialogService?.openDialogAndSubscribe(
```
Comment (niehuayang) [inline feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts]: **?** is redundant.

**Review by niehuayang — COMMENTED** [review]
just two same small comments

File: feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts (hunk 1 above)
Comment (scarai-sap) [reply feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts]: copy from below code, I guess yes, we can remove the ?


  createCustomer(): void {
    this.launchDialogService?.openDialogAndSubscribe(
      LAUNCH_CALLER.ASM_CREATE_CUSTOMER_FORM,
      this.addNewCustomerLink
    );
  }

Comment (niehuayang) [reply feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts]: yes, we can also remove it.

I guess that we fristly add 'launchDialogService' to custructor with a breakchange, then to haddle the break change, we had to difine it as 
```
constructor(
............
    protected launchDialogService**?**: LaunchDialogService
  ) {}
```
and in the major version, we don't remove all redundant code.

**Review by i53577 — DISMISSED** [review]

File: feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts (hunk 1 above)
Comment (scarai-sap) [reply feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts]: Yes, I guess so

**Review by niehuayang — DISMISSED** [review]

**Review by kimhw0630 — APPROVED** [review]

**Pull request merged**